packages/react/use-rect/src/useRect.tsx

```diff
@@ -1,19 +1,19 @@
 import * as React from 'react';
 import { observeElementRect } from '@radix-ui/rect';
-import { useLayoutEffect } from '@radix-ui/react-use-layout-effect';
 
 import type { Measurable } from '@radix-ui/rect';
 
 /**
  * Use this custom hook to get access to an element's rect (getBoundingClientRect)
  * and observe it along time.
  */
-function useRect(
+function useRect<T extends Measurable>(
   /** A reference to the element whose rect to observe */
-  refToObserve: React.RefObject<Measurable>
+  refToObserve: React.RefObject<T>
```

Comment:
Ah, I don't think i need this now. This was left over from when I was returning a `ref`, I wanted consumer to be able to do `useSize<HTMLDivElement>()` so that the returned ref would have the correct type.

I'll see if I can clean this up now.